spec/syntax/heredoc_spec.rb

```diff
@@ -4,28 +4,42 @@
 
 describe 'Heredoc syntax' do
   describe 'binary' do
-    it 'with multiline content' do
+    it 'doc with multiline content' do
       expect(<<~EOF).to include_elixir_syntax('elixirDocString', 'foo')
         @doc """
         foo
         """
       EOF
     end
 
-    it 'escapes quotes unless only preceded by whitespace' do
+    it 'doc with sigil_S' do
+      expect('~S(foo)').to include_elixir_syntax('elixirDocString', 'foo')
```

Comment:
this must not include `elixirDocString` since it doesn't begins with `@doc`, right?